src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -515,7 +515,7 @@ export default {
 		}
 
 		// EXP-MARS-143-Jul2022
-		this.shareCardLanguageVersion = this.$route.query?.scle;
+		this.shareCardLanguageVersion = this.$route.query?.utm_campaign;
```

Comment:
I think this should be gated with a check for `scle` within the utm_campaign param